packages/codegen-ui-react/lib/__tests__/studio-ui-codegen-react-forms.test.ts

```diff
@@ -653,7 +653,7 @@ describe('amplify form renderer tests', () => {
       expect(componentText).toMatchSnapshot();
     });
 
-    it.only('should render a update form with StorageField on non-array field', () => {
```

Comment:
curious if this is something we could lint for in the future

Reply:
That would be awesome. This is very easy to miss while looking at the diff.